test/unit/spec/util/support.js

```diff
@@ -33,6 +33,11 @@ describe('isSupported', () => {
       'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/81.0.4044.0 Safari/537.36',
       { runtime: {} }
     ],
+    [
```

Comment:
Can you add iPad user agent tests for both safari and chrome?

Reply:
There's already a test for iPad Safari UA test, so I'll add the Chrome one!